multiqc/utils/search_patterns.yaml

```diff
@@ -82,6 +82,32 @@ biobloomtools:
     num_lines: 2
 bismark/align:
     fn: '*_[SP]E_report.txt'
+biscuit/mapq:
```

Comment:
File search patterns should be unique for each _type_ of data. The way that the code is written currently makes it impossible for people to customise these search patterns if they modify their data files, as the module code expects certain file paths. So instead, each file type should have its own search pattern:

```yaml
biscuit/mapq_mapq_table: fn: '*_mapq_table.txt'
biscuit/mapq_strand_table: fn: '*_strand_table.txt'
biscuit/mapq_isize_score_table: fn: '*_isize_score_table.txt'
```

Reply:
Yes, now each file type has its own search pattern.